legate/core/operation.py

```diff
@@ -150,6 +161,7 @@ def execute(self):
 
     def get_tag(self, strategy, part):
         if strategy.is_key_part(part):
+            return 0
```

Comment:
This one is interesting. When creating a fused operation, if one of the stores has  a partition with tag=0, but another reference to the the same store w/ the same partition has tag=1 in the same fused op, the runtime will  complain that these 2 partitions are aliased, even though theoretically nothing is wrong. I will likely change this to to ignore tags when coalescing within a fused o[

Reply:
This sounds like an auto-partitioner issue: the auto-parallelizer marks a partition as the "key" partition used to distributed tasks across processors. And the auto-parallelizer conveys that information in the tag. With task fusion, the auto-parallelizer shouldn't pick a tag for a particular task, but for the whole window, so that the partition has the same tag throughout the window.